SU-SWS/stanford_bean_types
# Redesign cjw test

# READY FOR REVIEW

# Summary
- This PR is for the call-to-action blocks. This is for the 2.x branch, and _SoE needs the 2.x branch_. 

# Needed By (Date)
ASAPP (As soon as practically possible)

# Urgency
It's ready!

# Steps to Test
 
## To test:

1. Back up the database: drush sql-dump > ~/temp/jse-soe-new-<sitename>.sql
2. Download and enable field_group_link: drush dl field_group_link --destination=sites/default/modules/contrib ; drush en field_group_link -y
3. Revert feature: drush fr stanford_bean_types_call_to_action stanford_bean_types_call_to_action_administration -y --force
4. Enable modules: drush en stanford_bean_types_call_to_action stanford_bean_types_call_to_action_administration -y

### Configure layout

1. Navigate to Stanford Call to Action blocks admin/structure/block-types/manage/stanford-call-to-action/display
2. Enable Full Content
3. Navigate to full content
4. Move fields and field groups (Link group (Image style (Image, Icon), Title style(title)))"

### Test

1. Create a  **Call-to-Action** Bean
2. Verify the help text is displaying properly. See [SOE-1864](https://stanfordits.atlassian.net/browse/SOE-1864) and [SOE-1865](https://stanfordits.atlassian.net/browse/SOE-1865)
3. Navigate to **admin/manage/content** and verify it displays properly in the mange content view
4. Click on **Manage Call-to-Action Blocks** menu item
5. Verify it shows up on the Manage content page and that the page provides expected options
6. Place block and verify display and responsive behavior

### To run behat tests:

1. Check out the redesign-cjw-test branch (https://github.com/SU-SWS/linky_clicky/pull/170)
2. Navigate to `linky_clicky/sites/engineering-redesign`
3. Run 'live' tests on an Engineering redesigned site, e.g. `$ ../../bin/behat -p -s live features/engineering_stanford_bean_types`

# Affected Projects or Products
SoE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1864
- https://stanfordits.atlassian.net/browse/SOE-1865
- https://stanfordits.atlassian.net/browse/SOE-1651

- Anyone who should be notified? @boznik, @josephgknox 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

# Attachments
Here's how to adjust the _Manage Display_

![screen shot 2017-06-20 at 11 09 03](https://user-images.githubusercontent.com/284440/27348628-9874d77e-55a9-11e7-9dfc-a192455854a2.png)
